styles.css

```diff
@@ -0,0 +1,258 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    font-family: 'Assistant', sans-serif;
+    font-weight: regular;
+    background-color: white;
+}
+
+.container {
+    background-color: white;
+    margin: 0 auto;
+}
+
+.top-bar img {
+    max-width: 100%;
+    width: 100%;
+    background-color: #517171;
+}  
+
+#home-hero-desktop {
```

Comment:
Even though you only have one of these, I would still add a class to it and target it with the class, not the id. This will prevent CSS specificity issues later. I know it seems a little weird.